src/pages/AddItemPage.jsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from "react";
+import Button from "../components/Button";
+import Input from "../components/Input";
+import TagInput from "../components/TagInput";
+import TextArea from "../components/TextArea";
+import UploadImage from "../components/UploadImage";
+
+const AddItemPage = () => {
+  const [isDisable, setIsDisable] = useState(true);
+  const [name, setName] = useState("");
+  const [description, setDescription] = useState("");
+  const [price, setPrice] = useState("");
+  const [tag, setTag] = useState("");
+  useEffect(() => {
+    if (name && description && price && tag) setIsDisable(false);
+    else setIsDisable(true);
+  }, [name, description, price, tag]);
```

Comment:
💊 제안
아래처럼 작성하시는 것이 더 가독성에 좋을 것 같아요

```suggestion
  useEffect(() => {
    const isValidForm = name && description && price && tag;
    setIsDisable(!isValidForm);
  }, [name, description, price, tag]);
```

Reply:
❗️ 수정요청
등록 버튼 활성화 조건중 tag는 문자열이 아니라 문자열의 배열입니다~ 태그 추가 기능을 구현하시면서 이 부분도 같이 수정하시면 좋겠습니다~